rules/slack_rules/slack_privilege_changed_to_user.py

```diff
@@ -0,0 +1,25 @@
+from panther_base_helpers import deep_get, slack_alert_context
+
+
+def rule(event):
+    return event.get("action") == "role_change_to_user"
+
+
+def title(event):
+    username = deep_get(event, "entity", "user", "name", default="<unknown-entity>")
+    email = deep_get(event, "entity", "user", "email", default="<unknown-email>")
+
+    if event.get("action") == "role_change_to_user":
+        return f"{username}'s ({email}) role changed to User"
+
+    return f"Slack account changed on {username} ({email}), event: {event.get('action')}"
+
+
+def severity(event):
+    if event.get("action") != "role_change_to_user":
```

Comment:
it's not immediately clear to me that the severity could be changed with the definition of the `rule()` function. If the severity is static, I would recommend removing the severity function from python and instead setting the severity in the detection's yml file.

Reply:
Thanks for the feedback. Looking at the [docs](https://docs.panther.com/detections/writing-and-editing-detections/rules#rule-specification-reference) it looks like setting `Severity: Medium` in the yml is sufficient for this purpose?

Reply:
yep, that should work 👍